core/src/main/java/org/mule/transport/TransactedPollingMessageReceiver.java

```diff
@@ -137,14 +141,15 @@ public MuleEvent process() throws Exception
                     {
                         // this is not ideal, but jdbc receiver returns a list of maps, not List<MuleMessage>
                         List messages = getMessages();
+                        LinkedList<MuleEvent> results = new LinkedList<MuleEvent>();
```

Comment:
I need to use getLast() method and since it's not part of an interface or anything else makes sens to be a LinkedList